### DISCUSSION TITLE
Help restart nextcloud-aio-mastercontainer with different parameters

### DISCUSSION THREAD

**Discussion by gcala** [nextcloud/all-in-one#4944] [Questions] [answered]
Hi,
I had a working AIO instance installed following suggested instructions. Now I need to host an apache webserver so I followed instructions for [Reverse Proxy](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md).

I'm unable to restart the docker image. The previous container was created with this command:

```
sudo docker run \
--init \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
--env NEXTCLOUD_DATADIR="/mnt/ncdata" \
nextcloud/all-in-one:latest
```
The new command should be:

```
sudo docker run \
--init \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 8080:8080 \
--env APACHE_PORT=11000 \
--env APACHE_IP_BINDING=0.0.0.0 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
--env NEXTCLOUD_DATADIR="/mnt/ncdata" \
--env APACHE_IP_BINDING=127.0.0.1 \
nextcloud/all-in-one:latest
```
but obviously this command is for a new install.

I tried with:

```
sudo docker start \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
...
```

but got errors like "unknown flag: --sig-proxy", "unknown flag: --name"

What is the correct command to restart the container with new parameters, without wiping all current data?
Thanks

**Comment by szaimen** [comment, marked as answer]
> sudo docker run \
> --init \
> --sig-proxy=false \
> --name nextcloud-aio-mastercontainer \
> --restart always \
> --publish 8080:8080 \
> --env APACHE_PORT=11000 \
> --env APACHE_IP_BINDING=0.0.0.0 \
> --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
> --volume /var/run/docker.sock:/var/run/docker.sock:ro \
> --env NEXTCLOUD_DATADIR="/mnt/ncdata" \
> --env APACHE_IP_BINDING=127.0.0.1 \
> nextcloud/all-in-one:latest

looks good! You simply need to stop the mastercontainer first, remove it and recreate it with the above command

**Comment by gcala** [comment]
@szaimen Thanks, but I thought that deleting the mastercontainer did remove all databases, and data not saved in /mnt/ncdata...

**Reply by szaimen** [reply]
No it doesnt
